src/components/Personality/PersonalityCard.tsx

```diff
@@ -128,6 +128,14 @@ const PersonalityCard = ({
                                     type={ButtonType.blue}
                                     href={`${hrefBase ||
                                         "personality/"}${personality.slug}`}
+                                    style={{
+                                        width: "98px",
```

Comment:
Don't use fixed width and height on CSS changes unless is really necessary. This will not conform to the responsiveness of the website.